tools/project_manager/diff.php

```diff
@@ -116,7 +117,7 @@
 echo "<h2>$image_link</h2>\n";
 
 do_navigation($projectid, $image, $L_round_num, $R_round_num, 
-              $L_user_column_name, $L_user, $format);
+              $L_user_column_name, $L_user, $format, $L_text_column_name, $R_text_column_name, $only_nonempty_diffs);
```

Comment:
Please move `$L_user_column_name, $L_user, $format,` to the line above and even these out. Ditto in the `do_navigation()` definition too.